modules/sql-engine/src/integrationTest/sql/group1/types/time/time_parsing.test

```diff
@@ -28,84 +28,88 @@ SELECT '14:42:04.999000'::TIME(6)::VARCHAR
 ----
 14:42:04.999
 
-query T
+statement error: Invalid TIME value, '1'
 SELECT '1'::TIME
-----
-01:00:00
 
-query T
+statement error: Invalid TIME value, '11'
 SELECT '11'::TIME
-----
-11:00:00
 
-statement error
+statement error: Invalid TIME value, '11:'
 SELECT '11:'::TIME
 
-query T
+statement error: Invalid TIME value, '11:1'
 SELECT '11:1'::TIME
-----
-11:01:00
 
-statement error
+statement error: Invalid TIME value, '11:01'
+SELECT '11:01'::TIME
+
+statement error: Value of HOUR field is out of range in '25:42:04.500'
+SELECT '25:42:04.500'::TIME(3)
+
+statement error: Invalid TIME value
```

Comment:
Why do we have different message here than in previous statement?

Reply:
Thanks, fixed.